jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.html

```diff
@@ -19,11 +19,11 @@
     <tr data-field="root">
 
         <td class="kie-tablelist__cell">
-            <span data-field="name"></span>
+            <input data-field="name" type="text" required/>
         </td>
 
-        <td class="kie-tablelist__cell">
-            <span data-field="value"></span>
+        <td class="kie-tablelist__cell col-md-8">
```

Comment:
width: 70%; ?

Reply:
The row also have "Resolver type" and "Parameters" and delete button.
I think name and value totally is width:70.
Is it ok?